### ISSUE TITLE
Add capability to build with configuration file

### ISSUE THREAD

**Issue by arknable** [nanovms/ops-vscode#15]
Based on PR discussion here: https://github.com/nanovms/ops-vscode/pull/13#discussion_r645331480

**Comment by arknable** [comment]
@fabioDMFerreira @eyberg should be add this ?

**Comment by fabioDMFerreira** [comment]
Yes, I'm in favour of this. It's important that the user can build the image with a configuration file, so he can launch an instance from this image. 

We can achieve the same result with `Ops: Run`, but some users may want to use the 2 steps (build + launch instance from image).
